server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/OfficialDistribution.java

```diff
@@ -19,23 +19,25 @@
  */
 package org.sonar.server.platform.monitoring;
 
-import java.util.Map;
+import java.io.File;
+import org.sonar.api.platform.Server;
 import org.sonar.api.server.ServerSide;
-import org.sonar.server.platform.ws.InfoAction;
 
-/**
- * Any component that is involved in the information returned by the web service api/system/info
- */
 @ServerSide
-public interface Monitor {
-  /**
-   * Name of section in System Info page
-   */
-  String name();
+public class OfficialDistribution {
+  static final String BRANDING_FILE_PATH = "web/WEB-INF/classes/com/sonarsource/branding";
+
+  private final Server server;
+
+  public OfficialDistribution(Server server) {
+    this.server = server;
+  }
 
-  /**
-   * Type of attribute values must be supported by {@link org.sonar.api.utils.text.JsonWriter#valueObject(Object)}
-   * because of JSON export by {@link InfoAction}.
-   */
-  Map<String, Object> attributes();
+  public boolean check() {
+    // the dependency com.sonarsource:sonarsource-branding is shaded to webapp
+    // during release (see sonar-web pom)
+    File brandingFile = new File(server.getRootDir(), BRANDING_FILE_PATH);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getRootDir"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

Reply:
Done.